system/otherLoadTest/playlist.xml

```diff
@@ -314,24 +314,4 @@
 		</impls>
 		<platformRequirements>bits.64,^arch.arm</platformRequirements>
 	</test>
-	<!-- Temporarily excluded from z/Os due to : jdk11-zos/issues/342-->
-	<test>
-		<testCaseName>MixedLoadTest</testCaseName>
```

Comment:
`MixedLoadTest ` is removed. Is this intended?

Reply:
Yea, That target was redundant since we are now running the [time-based version of the same test](https://github.com/AdoptOpenJDK/openjdk-tests/blob/31a253b4a9a99f32aa370e7826ab6816e7371a29/system/otherLoadTest/playlist.xml#L21).

Reply:
We may need to update the issue as the test is replaced by `MiniMix_5m`.

Reply:
Instead of "replacement", we can look at it as as "conversion to time-based equivalent". 

So, Just like many other load tests are being converted to their `5m` equivalents in this PR, so has MixedLoadTest. The only difference being, for MixedLoadTest, conversion to its time-based equivalent(e.g. MiniMix_5m) happened in an [earlier PR](https://github.com/AdoptOpenJDK/openjdk-tests/pull/2107/files). This PR only contains the deletion of the old `MixedLoadTest`. I will add this explanation in the description of this PR.